docs/CONTRIBUTING.md

```diff
@@ -1,22 +0,0 @@
-# Contributing to xolotl 
-
-`xolotl` is far from feature complete, and your contributions are welcome. 
-
-## Reporting bugs
-
-* Is it a bug? Are you sure the bug persists after you run `transpile` and `compile`? 
-* Describe what the expected behaviour is, and what the actual behaviour was
-
-## Requesting features
-
-* Describe what you want
-* Describe why you want it
```

Comment:
-* List papers that describe this mechanism, or original research that describes the feature you want

Reply:
done